### ISSUE TITLE
Possible bug: "authors" is required but can be empty

### ISSUE THREAD

**Issue by ZedThree** [citation-file-format/citation-file-format#62]
This might just be restricted just to the Python schema validation, but passing an empty list as the `authors` key passed the schema validation (using pykwalifire). MVCE:

```python
from pykwalifire.core import Core
data = {
    'cff-version': "1.0.3",
    'message': "test data",
    'authors': [],
    'date-released': "2001-01-01",
    'title': "test title",
    'version': "1.2.3",
}
schema = "/path/to/CFF-Core/schema.yaml"
Core(source_data=data, schema_files=[schema]).validate()
```

**Comment by jspaaks** [comment]
Can we do something like JSON Schema's ``minItems``?:
http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.4.4

**Comment by jspaaks** [comment]
Looks like the stars just aligned

https://github.com/citation-file-format/citation-file-format/blob/7278290225e4ef10dc1289abe9a6d3f99cc80888/schema-sandbox/schema.json#L28-L38

**Comment by sdruskat** [comment]
Fixed in upcoming version 1.2.0.
